lib/user_interface_helper.rb

```diff
@@ -0,0 +1,46 @@
+require_relative "constants"
+
+module TicTacToe 
+  class UserInterfaceHelper
+
+    attr_accessor :input, :output
+
+    def initialize(input = STDIN, output = STDOUT)
+      @input = input
+      @output = output
+    end
+
+    def get_input
+      display(PROMPT)
+      @input.gets.chomp
+    end
+
+    def display(*messages)
+      messages.each do |message|
+        output.print message
+      end
+    end
+
+    def map_move(move, size) 
+      move -= 1
+      x = move / size
+      y = move % size
+      {x: x, y: y}
+    end
+
+    def draw_board(board, size)
+      mapped_board = ""
+
+      board.each_with_index do |row, i|
+        row.each_with_index do |cell, j|
+          cell_num = (i * size) + j + 1
+          mapped_board << (cell || cell_num.to_s).to_s.center(3)
+          mapped_board << "|" if j < size - 1
```

Comment:
Should/could I make these private methods even though `UI` calls these methods? Or are you suggesting having an intermediary method that `UI` calls and `UIHelper` calls the private methods?